be/src/vec/olap/vgeneric_iterators.cpp

```diff
@@ -75,7 +75,13 @@ Status VStatisticsIterator::next_batch(Block* block) {
         if (_push_down_agg_type_opt == TPushAggOp::COUNT) {
             size = std::min(_target_rows - _output_rows, MAX_ROW_SIZE_IN_COUNT);
             for (int i = 0; i < block->columns(); ++i) {
-                columns[i]->resize(size);
+                if (columns[i]->is_nullable()) {
```

Comment:
columns[i]->resize(size);
if (columns[i]->is_nullable()) {
// init nullmap here
// add comment to explain why we need init nullmap here.
}